bootstrap-salt.sh

```diff
@@ -4538,6 +4542,13 @@ install_amazon_linux_ami_git_deps() {
         yum -y install ca-certificates || return 1
     fi
 
+    PIP_EXE='pip'
+    if [ "$(echo "$GIT_REV" | egrep -o '2[0-9]{3}')" -ge 2016 ]; then
```

Comment:
Since `GIT_REV` could be any SHA1 commit hash, I think this statement could really make false positives...
Maybe it would be better just to check if Python 2.7 is available and go further with it?